test/functional/Java16andUp/src/org/openj9/test/foreignMemoryAccess/TestCloseScope0.java

```diff
@@ -83,12 +83,25 @@
 	@Test(expectedExceptions=java.lang.IllegalStateException.class)
 	public static void closeScopeDuringAccess() throws Throwable {
 		/* Reflect setup */
-		Class c = Class.forName("jdk.internal.foreign.MemoryScope");
-		Method createShared = c.getDeclaredMethod("createShared", new Class[] {Object.class, Runnable.class, Cleaner.class});
-		createShared.setAccessible(true);
-		Method close = c.getDeclaredMethod("close");
+		Class MemoryOrResourceScope;
```

Comment:
variable name: first letter should be lower case to be consistent
```suggestion
		Class memoryOrResourceScope;
```